examples/osqp_demo.c

```diff
@@ -1,5 +1,5 @@
 #include "osqp.h"
-
+#include <stdio.h>
```

Comment:
What is `stdio.h` needed for?

Reply:
artefact of debugging efforts.   Removed.